modules/home-manager/thunderbird.nix

```
@@ -31,7 +28,15 @@ in
     };
 
   config = lib.mkIf enable {
-    programs.thunderbird = {
+
+    assertions = [
+      {
+        assertion = correctVersion;
+        message = "catppuccin/nix thunderbird module requires at least version 25.05 of home-manager";
+      }
+    ];
```

Lets use our `catppuccinLib` to do the assertion.
```suggestion
    assertions = [ (catppuccinLib.assertMinimumVersion "25.05") ];
```

---

I was concerned about a misleading
```
error:
   Failed assertions:
   - pepperjack profile: `catppuccin/nix` requires at least version 25.05 of NixOS/home-manager
```
for stable users on upgrading this project, without knowing this is about thunderbird

---

Perhaps https://github.com/catppuccin/nix/blob/dc7e553e91c37cec5083ac5cfaff6a28565d1334/modules/lib/default.nix#L180 with `lib.versionAtLeast` may fix that.

---

Oh, nice, thanks !